nil pointer panic on add

#### Basic information

* [x] Version information (mark as appropiate):
  * [x] Master
  * [ ] Release candidate for next version
  * [ ] Latest stable version
  * [ ] An older version I should not be using
* [x] Type (mark as appropiate):
  * [x] Bug
  * [ ] Feature request
  * [ ] Enhancement
* [x] Operating system (mark as appropiate):
  * [x] Linux
  * [ ] macOS
  * [ ] Windows
  * [ ] Other: `which?`
* [x] Installation method (mark as appropiate):
  * [ ] Binaries from dist.ipfs.io
  * [ ] Built from sources
  * [x] Docker
  * [ ] Snap
  * [ ] Other: `which?`

#### Description

I encounter a nil pointer panic when adding a file or URL:
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x98973e]

goroutine 9 [running]:
main.main.func6.1(0xc00025a834, 0xc0000acc60, 0xc000066fc0)
        /go/src/github.com/ipfs/ipfs-cluster/cmd/ipfs-cluster-ctl/main.go:436 +0x35e
created by main.main.func6
        /go/src/github.com/ipfs/ipfs-cluster/cmd/ipfs-cluster-ctl/main.go:417 +0x5b3
```
The panic occurs here: https://github.com/ipfs/ipfs-cluster/blob/master/cmd/ipfs-cluster-ctl/main.go#L436
Last changed by: https://github.com/ipfs/ipfs-cluster/pull/637

--- a/cmd/ipfs-cluster-ctl/main.go
+++ b/cmd/ipfs-cluster-ctl/main.go
@@ -418,13 +418,13 @@ cluster "pin add".
 					defer wg.Done()
 
 					var buffered []*addedOutputQuiet
-					var lastBuf = make([]*addedOutputQuiet, 1, 1)
+					var lastBuf *addedOutputQuiet
 					var qq = c.Bool("quieter")
 					var q = c.Bool("quiet") || qq
 					var bufferResults = c.Bool("no-stream")
 					for v := range out {
 						added := &addedOutputQuiet{v, q}
-						lastBuf[0] = added
+						lastBuf = added
 						if bufferResults {
 							buffered = append(buffered, added)
 							continue
@@ -433,18 +433,18 @@ cluster "pin add".
 							formatResponse(c, added, nil)
 						}
 					}
-					if lastBuf[0].added == nil {
+					if lastBuf == nil || lastBuf.added == nil {
 						return // no elements at all
 					}
 					if bufferResults { // we buffered.
 						if qq { // [last elem]
-							formatResponse(c, lastBuf, nil)
+							formatResponse(c, []*addedOutputQuiet{lastBuf}, nil)
 							return
 						}
 						// [all elems]
 						formatResponse(c, buffered, nil)
 					} else if qq { // we already printed unless Quieter
-						formatResponse(c, lastBuf[0], nil)
+						formatResponse(c, lastBuf, nil)
 						return
 					}
 				}()

